flow-model-generator/modules/flow-model-generator-ls-extension/src/main/java/io/ballerina/flowmodelgenerator/extension/ExpressionEditorService.java

```diff
@@ -251,6 +253,32 @@ public CompletableFuture<FunctionCallTemplateResponse> functionCallTemplate(Func
         });
     }
 
+    @JsonRequest
+    public CompletableFuture<SuccessResponse> moduleSnippet(ModuleSnippetRequest request) {
```

Comment:
IINM this API is specific to applying import statements, so any reason not to reflect that in the API name and the request record name? Are we planning to re-use the same API for extended purposes in the future?

Reply:
Thanks for noting this. The last commit was not pushed